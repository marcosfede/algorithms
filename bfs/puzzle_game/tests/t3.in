3

4 1 2
9 3 5
6 7 8

4 3 2
7 6 5
1 9 8

9 5 1
2 8 7
6 4 3