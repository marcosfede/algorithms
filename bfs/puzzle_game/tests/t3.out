24
-1
44